docs/Tutorials/GpsTutorial/Tutorial.md

```diff
@@ -971,7 +971,7 @@ separate terminal run the following:
 
 ```shell
 cd fprime/GpsApp
-scp bin/arm-linux-gnueabihf/GpsApp pi@<raspberry pi IP>
+scp build-artifacts/arm-linux-gnueabihf/bin/GpsApp pi@<raspberry pi IP>
```

Comment:
Is this build directory correct?

Reply:
Yep, I previously tested to confirm. We use the platform from cmake, which isn't necessarily the same name as the toolchain name passed to fprime.